### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by voruti**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,33 +1,26 @@
...
- - Es wird mit Vaadin + Spring Boot eine Webseite verfügbar gemacht, mit der man häufige BWL Rechnungen effizent lösen kann
+## Was macht unser Tool?
+- Es wird Port 8080 für einen Tomcat Webserver geöffnet
+- Es wird mit Vaadin + Spring Boot eine Webseite verfügbar gemacht, mit der man häufige BWL Rechnungen effizent lösen kann
 
-<H2>Richtig in der IDE zum laufen bringen</H2>
+## Richtig in der IDE zum laufen bringen
+1. Die pom.xml mit IntelliJ oder Eclipse öffnen
+2. Abwarten bis alle Depencencies heruntergeladen wurden
+3. Am Besten mit Java 11 starten
```
Comment (LReg) [inline README.md:11]: Sure 11 is better? It was build from the vaadin website for 15

**Review by LReg — COMMENTED** [review]
Not sure about the Java version

File: README.md (hunk 1 above)
Comment (voruti) [reply README.md:11]: From pom.xml:
```xml
<properties>
    <java.version>11</java.version>
```
Maven builds this project with Java 11.

I don't care which Java version is used, but it should be consistent.

Comment (LReg) [reply README.md:11]: Okay, akward that it is generated like this...

**Pull request merged**